.github/workflows/integration.yml

```diff
@@ -9,11 +9,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cluster: ["opendistro", "opensearch"]
+        cluster: ["opensearch"]
```

Comment:
shall we split this into two jobs to keep it simple? ( opendistro and opensearch )

Reply:
Sure I can split it up.